nevergrad/optimization/recastlib.py

```diff
@@ -28,11 +29,23 @@ def __init__(
     ) -> None:
         super().__init__(parametrization, budget=budget, num_workers=num_workers)
         self.multirun = 1  # work in progress
+        self._normalizer: tp.Any = None
         self.initial_guess: tp.Optional[tp.ArrayLike] = None
         # configuration
-        assert method in ["Nelder-Mead", "COBYLA", "SLSQP", "Powell"], f"Unknown method '{method}'"
+        assert method in [
+            "CmaFmin2",
+            "Nelder-Mead",
+            "COBYLA",
+            "SLSQP",
+            "Powell",
+        ], f"Unknown method '{method}'"
         self.method = method
         self.random_restart = random_restart
+        # The following line rescales to [0, 1] if fully bounded.
+        if method == "CmaFmin2" and p.helpers.Normalizer(self.parametrization).fully_bounded:
+            self._normalizer = p.helpers.Normalizer(self.parametrization)
+        else:
+            self._normalizer = None
```

Comment:
Initialization is heavy so better compute it only once
```suggestion
        self._normalizer: tp.Optional[p.helpers.Normalizer] = None
        if method == "CmaFmin2":
             normalizer = p.helpers.Normalizer(self.parametrization)
             if normalizer.fully_bounded:
            self._normalizer = normalizer
```